pkgs/tools/misc/oh-my-posh/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, getThemes ? false
```

Comment:
Why don't we want the default themes?

Reply:
According to docs here: https://ohmyposh.dev/docs/installation/customize  
oh-my-posh can generate specific shell scripts from json format theme,  
source the generated scripts to get theme.

That theme folder is collection listing here: https://ohmyposh.dev/docs/themes
Access by using `${generated.oh-my-posh.src}/themes/kali.omp.json` is also same.

Reply:
Those themes are in total not bigger than 8MB. We can safely copy them to the output all the time and remove that flag.